src/ffi/extra/library_module_system_lib.cc

```diff
@@ -125,6 +125,14 @@ class SystemLibModuleRegistry {
   Map<String, ffi::Module> lib_map_;
 };
 
+static int test_symbol_add_one(void*, const TVMFFIAny* args, int32_t num_args, TVMFFIAny* ret) {
+  TVM_FFI_SAFE_CALL_BEGIN();
```

Comment:
let us avoid this in the lib, maybe to testing? or in this case better to skip

Reply:
moved to `testing.cc` per discussion